benchmarks/src/main/java/io/druid/benchmark/query/TopNBenchmark.java

```diff
@@ -372,7 +372,7 @@ public void queryMultiQueryableIndex(Blackhole blackhole)
 
     Sequence<Result<TopNResultValue>> queryResult = theRunner.run(
         QueryPlus.wrap(query),
-        Maps.<String, Object>newHashMap()
+        Maps.newHashMap()
```

Comment:
Did you remove redundant type arguments in the whole codebase? If so, you could try to change the level of the corresponding IntelliJ inspection ("Redundant type arguments") to ERROR.

Reply:
Added.